services/monitor/instances/monitor_gcf_env_secrets/monitor_gcf_env_secrets.rego

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+package templates.gcp.GCPCloudfunctionEnvSecretConstraintV1
+
+import data.validator.gcp.lib as lib
+
+deny[{
+	"msg": message,
+	"details": metadata,
+}] {
+	constraint := input.constraint
+	lib.get_constraint_params(constraint, params)
+
+	name_whitelist := lib.get_default(params, "name_whitelist", [])
+	envname_contains := lib.get_default(params, "name_contains", [])
+	envvalue_contains := lib.get_default(params, "value_contains", [])
```

Comment:
The check on name contains is case sensitive, so with the constraint set to "PASS" it detects a variable named PASSWORD but it did not detected the variable named password.
would you make it case insensitive ?

Reply:
Done